src/hpc/tls.c

```diff
@@ -19,9 +19,13 @@ __thread ThreadLocalStorage *TLSInstance;
 
 void InitializeTLS(void)
 {
-#ifdef HAVE_NATIVE_TLS
+#ifdef USE_NATIVE_TLS
     // FIXME: is this the right place to do this?
     TLSInstance = &(TLSGAPtate.tls);
+#endif
+#ifdef USE_PTHREAD_TLS
+    void *AllocateTLS(void);
```

Comment:
No, just: no! Don't declare external references inline like that. Add them properly to a header, or if such a header already exists (as in this case: `src/hpc/thread.h`), just #include that header.

Reply:
Oops, fixed.